tests/chains/Ethereum/SignerTests.cpp

```diff
@@ -284,7 +284,7 @@ TEST(EthereumSigner, EIP4337_ERC1155_Transfer_Account_Deployed) {
 
 TEST(EthereumSigner, SignatureBreakdownNoEip155) {
     const auto key = PrivateKey(parse_hex("f9fb27c90dcaa5631f373330eeef62ae7931587a19bd8215d0c2addf28e439c8"));
-    const auto hash = parse_hex("0xf86a808509c7652400830130b9946b175474e89094c44da98b954eedeac495271d0f80b844a9059cbb0000000000000000000000005322b34c88ed0691971bf52a7047448f0f4efc840000000000000000000000000000000000000000000000001bc16d674ec80000808080");
+    const auto hash = parse_hex("0xf86a808509c7652400830130b9946b175474e89094c44da98b954eedeac49527");
```

Comment:
hmm, how did this test work before?

Reply:
[ecdsa_sign_digest](https://github.com/trustwallet/wallet-core/blob/423f0e34725f69c0a9d535e1a32534c99682edea/trezor-crypto/crypto/ecdsa.c#L679-L782) takes the first 32 bytes of the `digest` ignoring the remaining part.
https://github.com/trustwallet/wallet-core/blob/423f0e34725f69c0a9d535e1a32534c99682edea/trezor-crypto/crypto/ecdsa.c#L693